.github/workflows/_build.yml

```diff
@@ -54,10 +60,26 @@ jobs:
           ./gradlew assemble${{ env.flavour }}${{ env.config }} -PdisablePreDex
 
       - name: Tokens check
+        if: ${{ github.event_name != 'pull_request' || github.actor != 'dependabot[bot]' }} || ${{ github.event_name == 'pull_request' && github.actor == 'dependabot[bot]' }} && !contains(steps.dependabot-metadata.outputs.dependency-names, 'bpk-')
         run: |
           ./gradlew generateTokens -PdisablePreDex
           ./scripts/check-no-changes.sh
 
+      - name: Token update
+        if: ${{ github.event_name == 'pull_request' && github.actor == 'dependabot[bot]' }} && contains(steps.dependabot-metadata.outputs.dependency-names, 'bpk-')
```

Comment:
and this is the opposite - we only want to run this for the backpack dependency update. in that case we just commit the changes. i tried testing this in the currently open dependabot PR, but then the actor is me, not dependabot. will merge and then recreate the dependabot PR to test